### ISSUE TITLE
Test that Font Awesome glyphs are not announced by screen readers

### ISSUE THREAD

**Issue by kimisgold** [omeka/omeka-s#2371]
Omeka S uses Font Awesome icons throughout its interface. Despite Font Awesome using characters from the Unicode Private Use Area, screen readers like Voiceover for mac OS still read out unknown characters. Omeka S is using Font Awesome 5.15.4, the last anticipated version of Font Awesome 5; with that in mind, we're applying our own usage of [alternative text for pseudo elements](https://developer.mozilla.org/en-US/docs/Web/CSS/content#alternative_text_string_counter), setting them as blank strings that don't get announced by screen readers.

Test on the `hide-iconfont-pseudo-elements` branch that font icons appear as expected throughout the admin interface and don't get read by screen readers.

**Comment by aviannamiller** [comment]
That's fixed! The separator is accompanied by an announcement ("middle dot", at least on VoiceOver), though that seems okay for now if you would like to close here.
